### ISSUE TITLE
Arabic custom font

### ISSUE THREAD

**Issue by Fshamri** [delight-im/Android-AdvancedWebView#29]
Hi,
Android webView can render custom english fonts without any issue. this is not the case in custom Arabic font. here the webView looks for the default system Arabic font and renders it. it ignores any defined custom font.
 Can you please bring a solution for this issue in your Android-AdvanceWebView?

Thanks

**Comment by mwaclawek** [comment]
Thanks for your question!

Luckily, due to the way we have extended the original `WebView` class, you can use any solution for our `AdvancedWebView` that works for the original `WebView`.

Unfortunately, it seems others are having the same problem that you have, and there may not be a solution yet:

 * http://stackoverflow.com/questions/15631713/android-webview-with-arabic-content-doesnt-load-custom-font
 * http://stackoverflow.com/questions/19048430/arabic-font-issue-in-the-android-webview
 * http://stackoverflow.com/questions/32013293/%D9%90arabic-custom-font-in-android-webview

So, in short, this problem isn't related to our library. We hope you'll find a solution to this general problem with the `WebView` classes. You're welcome to post your solution here when you find one :)
